### DISCUSSION TITLE
建议增加转发stun流量的端口

### DISCUSSION THREAD

**Discussion by ghost** [immortalwrt/homeproxy#175] [Q&A] [answered]
建议增加转发stun流量的端口，stun流量没有被转发，webrtc泄露真实IP，自己修改配置文件增加端口软件就无法启动

**Comment by 1715173329** [comment]
那是你配置问题。

**Comment by ghost** [comment]
我使用代理所有端口，webrtc就显示的代理Ip，但我又不能使用这个模式，常用端口上也没有stun的端口

**Comment by muink** [comment, marked as answer]
3478 19302 加入 common_port, 如果有用到其他 stun 端口, 也加进去
但是会影响通过 stun 检查 NAT 行为的程序的结果准确性

**Reply by ghost** [reply]
> 3478 19302 加入 common_port, 如果有用到其他 stun 端口, 也加进去 但是会影响通过 stun 检查 NAT 行为的程序的结果准确性

感谢回复，ios和windows已经不泄露了，仅android还泄露ipv6，不过已经足够了
